File: review_handler/review_handler.Core/Entities/Actor.cs
namespace review_handler.Core.Entities
{
    public class Actor : Person
    {
        public List<Movie> Filmography { get; set; }
        public List<Award> Awards { get; set; }

        public Actor(string firstName, string lastName, DateTime dateOfBirth, string biography, string nationality, List<Movie> filmography, List<Award> awards)
            : base(firstName, lastName, dateOfBirth, biography, nationality)
        {
            this.Filmography = filmography;
            this.Awards = awards;
        }

        public Actor() { }
    }
}

File: movie_handler/src/main/java/com/example/model/Actor.java
package com.example.model;

import com.example.searchService.Search;
import jakarta.persistence.Entity;

import java.util.Date;
import java.util.List;
import java.util.UUID;

@Entity
public class Actor extends Person {
    private List<Movie> filmography;
    private List<Award> awards;

    public Actor(String firstName, String lastName, Date dateOfBirth, String nationality, String biography, UUID profileImage, List<Movie> filmography, List<Award> awards) {
        super(firstName, lastName, dateOfBirth, nationality, biography, profileImage);
        this.filmography = filmography;
        this.awards = awards;
    }

    public Actor() {

    }

    public List<Movie> getFilmography() {
        return filmography;
    }

    public void setFilmography(List<Movie> filmography) {
        this.filmography = filmography;
    }

    public List<Award> getAwards() {
        return awards;
    }

    public void setAwards(List<Award> awards) {
        this.awards = awards;
    }

    @Override
    public void accept(Search searchObject) {

    }
}
